wsagent/che-core-api-debug-shared/src/main/java/org/eclipse/che/api/debug/shared/model/ThreadDump.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.debug.shared.model;
+
+import java.util.List;
+import org.eclipse.che.commons.annotation.Nullable;
+
+/** @author Anatoliy Bazko */
+public interface ThreadDump {
```

Comment:
I think this is not correct to use name `ThreadDump` here, because
> A thread dump is a snapshot of the state of all threads that are part of the process.

It looks like a `ThreadDumpItem` or so.